src/meadowflow/scheduler.py

```diff
@@ -522,10 +517,8 @@ def manual_run(
         if job_name not in self._jobs:
             raise ValueError(f"Unknown job: {job_name}")
         job = self._jobs[job_name]
-        self._event_loop.call_soon_threadsafe(
-            lambda: self._event_loop.create_task(
-                self._run_action(job, Actions.run, overrides)
-            )
+        asyncio.get_running_loop().call_soon(
+            lambda: asyncio.create_task(self._run_action(job, Actions.run, overrides))
         )
 
     async def manual_run_on_event_loop(
```

Comment:
Same thing here, I think `manual_run` and `manual_run_on_event_loop` can be turned into a single function now